addOns/client/src/main/java/org/zaproxy/addon/client/ExtensionClientIntegration.java

```diff
@@ -182,4 +185,16 @@ public void sessionModeChanged(Mode mode) {
             // Ignore
         }
     }
+
+    public void addReportedZestStatement(JSONObject json) throws Exception {
+        clientHandler.addZestStatementUtil(json);
+    }
+
+    public void setClientRecorderHelper(ZestClientRecordingHelper clientHandler) {
```

Comment:
Why is it using the wrapper instead of `ClientZestInterface`? I don't think the wrapper is needed (at least not now).

Reply:
No worries, we all do them :)

Reply:
The class `ZestClientRecordingHelper` could be removed and set the method reference.